supabase/migrations/20260826061707_org_billing_column_guard.sql

```
@@ -0,0 +1,64 @@
+-- Block direct PostgREST writes to orgs.customer_id unless the caller has org.update_billing.
```

SQLFluff (4.3.0)

[error] Line is too long (94 > 80).

(LT05)